setup.py

```diff
@@ -39,6 +38,7 @@
 DEV_REQUIRES = [
     'ipdb',
     'ipython',
+    'pytest-sugar',
```

Comment:
Ummmm, I don't quite agree with this. This section is for all the packages need for development only. You should make a TEST_OPTIONAL list and then add it there.

Reply:
We should probably document what each of these sections is for as well for people in the future adding dependencies

Reply:
I thought the naming and categorization was pretty obvious given we have a separate tests and dev section. But you are right, some comments would be better :)